OnlineStore.BLL/Exceptions/ModelValidationFailedException.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace OnlineStore.BLL.Exceptions
+{
```

Comment:
there is no need to throw exception on model validation fail, so you wont need such class